packages/web/src/common/services/local-storage/LocalStorage.ts

```diff
@@ -0,0 +1,76 @@
+import { User } from '@audius/common'
+import { CURRENT_USER_EXISTS_LOCAL_STORAGE_KEY } from '@audius/sdk/dist/core'
+
+// TODO: the following should come from @audius/libs/dist/core when
+// discoveryProvider/constants is migrated to typescript.
+const DISCOVERY_PROVIDER_TIMESTAMP = '@audius/libs:discovery-node-timestamp'
+
+const AUDIUS_ACCOUNT_KEY = '@audius/account'
+const AUDIUS_ACCOUNT_USER_KEY = '@audius/audius-user'
+
+type LocalStorageType = {
+  getItem: (key: string) => Promise<string | null> | string | null
+  setItem: (key: string, value: string) => Promise<void> | void
+  removeItem: (key: string) => Promise<void> | void
+}
+
+type LocalStorageConfig = {
+  localStorage: LocalStorageType
+}
+
+export class LocalStorage {
```

Comment:
Did you make this a class instead of a function for consistency with the other services?

Reply:
yeah i guess so, happy to make a function if we think it makes more sense though.